main.go

```diff
@@ -46,13 +47,13 @@ var onCompress = js.FuncOf(func(this js.Value, args []js.Value) interface{} {
 
 	for i := 1; i <= count; i++ {
 		Log("Processing page no:", i)
-		imageObjNrs := ctx.ImageObjNrs(i)
+		imageObjNrs := pdfcpu.ImageObjNrs(ctx, i)
 		Log("Images on page:", len(imageObjNrs))
-		images, err := ctx.ExtractPageImages(i, false)
+		images, err := pdfcpu.ExtractPageImages(ctx, i, false)
```

Comment:
ExtractPageImages returns a map instead of an array. Please update the lines below.